pydicom/config.py

```diff
@@ -110,16 +106,30 @@ def get_pixeldata(ds):
 """
 
 
-def debug(debug_on=True):
-    """Turn debugging of DICOM file reading and writing on or off.
+def debug(debug_on=True, default_handler=True):
+    """Turn on/off debugging of DICOM file reading and writing.
+
     When debugging is on, file location and details about the
     elements read at that location are logged to the 'pydicom'
     logger using python's logging module.
 
-    :param debug_on: True (default) to turn on debugging,
-    False to turn off.
+    Parameters
+    ----------
+    debug_on : bool, optional
+        If True (default) then turn on debugging, False to turn off.
+    default_handler : bool, optional
+        If True (default) then use ``logging.StreamHandler()`` as the handler
+        for log messages.
     """
     global logger, debugging
+    logger = logging.getLogger('pydicom')
```

Comment:
Why you have to get the logger again here? Can it change to another instance elsewhere?

Reply:
I was writing why I needed it and then realised what the cause was (the earlier one liner which added the NullHandler() returns None). 

Will fix.